[2022-06-28 14:16:43,708] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test.training_model_A scheduled__2022-06-27T00:00:00+00:00 [queued]>
[2022-06-28 14:16:43,721] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test.training_model_A scheduled__2022-06-27T00:00:00+00:00 [queued]>
[2022-06-28 14:16:43,721] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-28 14:16:43,721] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-28 14:16:43,721] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-28 14:16:43,749] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): training_model_A> on 2022-06-27 00:00:00+00:00
[2022-06-28 14:16:43,764] {standard_task_runner.py:52} INFO - Started process 76 to run task
[2022-06-28 14:16:43,770] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'training_model_A', 'scheduled__2022-06-27T00:00:00+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/second_2.py', '--cfg-path', '/tmp/tmpdiir90lc', '--error-file', '/tmp/tmpdk3dy_hf']
[2022-06-28 14:16:43,770] {standard_task_runner.py:80} INFO - Job 206: Subtask training_model_A
[2022-06-28 14:16:43,899] {logging_mixin.py:109} INFO - Running <TaskInstance: test.training_model_A scheduled__2022-06-27T00:00:00+00:00 [running]> on host ca68cb5b3870
[2022-06-28 14:16:44,001] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-06-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-27T00:00:00+00:00
[2022-06-28 14:16:44,001] {python.py:175} INFO - Done. Returned value was: 7
[2022-06-28 14:16:44,088] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=test, task_id=training_model_A, execution_date=20220627T000000, start_date=20220628T141643, end_date=20220628T141644
[2022-06-28 14:16:44,118] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-28 14:16:44,166] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
